packages/react/src/components/Tree/Tree.tsx

```diff
@@ -0,0 +1,331 @@
+import * as customPropTypes from '@stardust-ui/react-proptypes'
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+import { Ref } from '@stardust-ui/react-component-ref'
+
+import TreeItem, { TreeItemProps } from './TreeItem'
+import {
+  childrenExist,
+  commonPropTypes,
+  createShorthandFactory,
+  UIComponentProps,
+  ChildrenComponentProps,
+  rtlTextContainer,
+  applyAccessibilityKeyHandlers,
+  AutoControlledComponent,
+} from '../../lib'
+import {
+  ShorthandRenderFunction,
+  WithAsProp,
+  withSafeTypeForAs,
+  ShorthandCollection,
+  ShorthandValue,
+} from '../../types'
+import { Accessibility } from '../../lib/accessibility/types'
+import { treeBehavior } from '../../lib/accessibility'
+import { getNextElement } from '../../lib/accessibility/FocusZone/focusUtilities'
+
+export interface TreeSlotClassNames {
+  item: string
+}
+
+export interface TreeProps extends UIComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility
+
+  /** Map with the subtrees and information related to their open/closed state. */
```

Comment:
```suggestion
  /** Ids of opened items. */
```

Not sure that my suggestion is the best, but description should be updated.